modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsAutoIncrement.testcase

```diff
@@ -101,6 +101,99 @@ definition {
 		}
 	}
 
+	@description = "LPS-169223 - Verify after importing an object entry with a modified auto increment field value, new object entries increment based off of the modified value"
+	@priority = 4
+	test CanIncrementAfterImportingObjectEntry {
+		task ("Given an object definition called Shoes that contains an auto-increment field and a text field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Shoe",
+				objectName = "Shoe",
+				pluralLabelName = "Shoes");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Color",
+				fieldName = "color",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "Shoe");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "AutoIncrement",
+				fieldLabelName = "Identification",
+				fieldName = "identification",
+				fieldType = "String",
+				initialValue = 001,
+				isRequired = "false",
+				objectName = "Shoe",
+				prefix = "",
+				suffix = "-Sneakers");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "Shoe");
+		}
+
+		task ("and Given an object entry") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				ercName = "shoe001",
+				fieldName = "color",
+				objectName = "Shoe",
+				value = "Blue");
+		}
+
+		task ("and Given the object entry is exported") {
+			ImportExport.exportAndUnzipFile(
+				entityType = "C_Shoe (v1_0 - Liferay Object REST)",
+				exportFileFormat = "JSON");
+		}
+
+		task ("and Given the auto increment field value and the ERC are modified in the JSON file") {
```

Comment:
Hey @andreefarias, since we're changing the text field and not the ERC, can you update the wording here? Just replace `ERC` with `text field value`

Reply:
Ok, I'll update.

Reply:
Wait... but I'm changing the ERC and not the text field.

ERC = `FileUtil.replaceStringInFile(${filePath}, "shoe001", "shoe020");`
Auto increment = `FileUtil.replaceStringInFile(${filePath}, "001-Sneakers", "020-Sneakers");`